lib/oli/utils.ex

```diff
@@ -3,8 +3,8 @@ defmodule Oli.Utils do
 
   import Ecto.Changeset
 
-  @urlRegex ~r/(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/i
-  @emailRegex ~r/^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/
```

Comment:
Not sure the exact requirements here, but a regex for valid RFC 5322 email address validations is a lot more complicated than this.

I think this will allow some addresses that it shouldn't like: `first..last@gmail.com`  (consecutive . are not allowed)
and it will reject some addresses that it should allow like `first+last@gmail.com` (+ should be allowed)

Reply:
Awesome, rejecting valid emails was my primary worry, so sounds good.